docs/source/services/timelock_service/installation.rst

```diff
@@ -50,3 +70,39 @@ Timelock Server Installation
               "healthy": true
           }
       }
+
+Add client(s) to Timelock
+-------------------------
+
```

Comment:
Would it make sense for us to include another tip for internal consumers to use the standard internal procedure for this?